collectionFunctions.php

```diff
@@ -63,18 +63,19 @@ function addStyle(): String {
  * @param PDO $database
  */
 function checkNew(PDO $database) {
-    if (isset($_POST['name'], $_POST['type1'], $_POST['hp'], $_POST['attack'], $_POST['defense'], $_POST['spAttack'], $_POST['spDefense'], $_POST['speed'])) {
+    if (isset($_POST['name'], $_POST['type1'], $_POST['hp'], $_POST['attack'], $_POST['defense'], $_POST['spAttack'], $_POST['spDefense'], $_POST['speed'])
```

Comment:
This logic works for preventing inserting strings into the DB, but the error message never actually shows